shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -590,7 +591,14 @@ private void configureStatusBarForFullscreenFlutterExperience() {
       Window window = getWindow();
       window.addFlags(WindowManager.LayoutParams.FLAG_DRAWS_SYSTEM_BAR_BACKGROUNDS);
       window.setStatusBarColor(0x40000000);
-      window.getDecorView().setSystemUiVisibility(PlatformPlugin.DEFAULT_SYSTEM_UI);
+      WindowCompat.setDecorFitsSystemWindows(window, false);
+      if (Build.VERSION.SDK_INT < 30) {
+        // This ensures that the navigation bar is not hidden for APIs 19-30,
```

Comment:
is it 19-29 (inclusive) or 19-30 (inclusive)?

Reply:
It is 19-29 inclusive. I will update the comment to better reflect that!